backend/src/ast/converters/scratch/scratch-ast-converter.ts

```diff
@@ -178,7 +178,7 @@ const partitionRootBlocks = (
     } else if (isDefinitionBlock(block)) {
       procedureDefinitionRoots.push(block);
     } else {
-      throw new Error("Unexpected top-level block type");
+      // ignore stacks starting with non-hat blocks
```

Comment:
Could we keep somehow dangling stacks around ? 
To me, this is the Scratch equivalent of having a function that's never called, or an `if (false) { ... }` in the code. It might be good to keep a representation of that somehow.

Reply:
https://classroomrt.atlassian.net/browse/CRT-52